packages/sdk/review/teamsfx.api.md

```diff
@@ -494,6 +523,8 @@ export interface TeamsFxBotSsoCommandHandler {
 
 // @public
 export class TeamsUserCredential implements TokenCredential {
+    // Warning: (ae-forgotten-export) The symbol "TeamsUserCredentialAuthConfig" needs to be exported by the entry point index.d.ts
```

Comment:
You might forget to export `TeamsUserCredentialAuthConfig`.

Reply:
Oh, thanks for your reminding